src/autopas/containers/cellPairTraversals/SlicedBasedTraversal.h

```diff
@@ -220,15 +231,15 @@ void SlicedBasedTraversal<ParticleCell, PairwiseFunctor, dataLayout, useNewton3>
     // all but the first slice need to lock their starting layers.
     if (slice > 0) {
       for (unsigned long i = 0ul; i < _overlapLongestAxis; i++) {
-        locks[((slice - 1) * _overlapLongestAxis) + i].lock();
+        _locks[((slice - 1) * _overlapLongestAxis) + i].lock();
       }
     }
     const auto lastLayer = myStartArray[_dimsPerLength[0]] + _sliceThickness[slice];
     for (unsigned long dimSlice = myStartArray[_dimsPerLength[0]]; dimSlice < lastLayer; ++dimSlice) {
       // at the last layers request lock for the starting layer of the next
       // slice. Does not apply for the last slice.
       if (slice != numSlices - 1 && dimSlice >= lastLayer - _overlapLongestAxis) {
```

Comment:
```suggestion
      if (slice != numSlices - 1 and dimSlice >= lastLayer - _overlapLongestAxis) {
```
We try to become consistent with those throughout the code. Please replace those whenever you see them.
Also applies to `or` instead of `||`.